client/src/helpers/APICalls/updateProfile.ts

```diff
@@ -0,0 +1,16 @@
+import { ProfileApiData } from '../../interface/ProfileApiData';
+import { FetchOptions } from '../../interface/FetchOptions';
+
+const updateProfile = async (values: ProfileApiData): Promise<ProfileApiData> => {
+  console.log('The values are: ', JSON.stringify({ values }));
+  const fetchOptions: FetchOptions = {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify(values),
+    credentials: 'include',
+  };
+  return await fetch(`/profile/updateProfile`, fetchOptions).then((res) => res.json());
+  // .catch(() => ({ error: { message: 'Unable to connect to server. Please try again' } }));
```

Comment:
This could be either brought back or removed, but we should be catching for any errors that occur

Reply:
OK